github/org_custom_roles.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// OrgainizationCustomRepoRoles represents custom repository roles available in specified organization.
+type OrgainizationCustomRepoRoles struct {
+	TotalCount      *int               `json:"total_count,omitempty"`
+	CustomRepoRoles []*CustomRepoRoles `json:"custom_roles,omitempty"`
+}
+
+// CustomRepoRoles represents custom repository roles for an organization.
+// See https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-peoples-access-to-your-organization-with-roles/managing-custom-repository-roles-for-an-organization
+// for more information.
+type CustomRepoRoles struct {
+	ID   *int64  `json:"id,omitempty"`
+	Name *string `json:"name,omitempty"`
+}
+
+// ListCustomRepoRoles lists the custom repository roles available in this organization.
+// In order to see custom repository roles in an organization, the authenticated user must be an organization owner.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/orgs#custom-repository-roles
+func (s *OrganizationsService) ListCustomRepoRoles(ctx context.Context, org string) (*OrgainizationCustomRepoRoles, *Response, error) {
```

Comment:
Were you able to test this endpoint out?  I ask because I don't think it will work as written.
According to the docs that I referenced above, it returns a list, not a single item.

```suggestion
func (s *OrganizationsService) ListCustomRepoRoles(ctx context.Context, org string) ([]*OrgainizationCustomRepoRoles, *Response, error) {
```

Reply:
WHUPS!  I apologize... I misread the docs.  I think you are correct.  Please disregard.